<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmulationSettingsWidget</class>
 <widget class="QWidget" name="EmulationSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Speed Control</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Emulation Speed:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="emulationSpeed"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Fast Forward Speed:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="fastForwardSpeed"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Turbo Speed:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="turboSpeed"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Rewind/Runahead</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="rewindEnable">
        <property name="text">
         <string>Enable Rewinding</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Rewind Save Frequency:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="rewindSaveFrequency">
        <property name="suffix">
         <string> Seconds</string>
        </property>
        <property name="maximum">
         <double>3600.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Rewind Buffer Size:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="rewindSaveSlots">
        <property name="suffix">
         <string> Frames</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Runahead:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="runaheadFrames">
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 Frame</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 Frames</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3 Frames</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4 Frames</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 Frames</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6 Frames</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7 Frames</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8 Frames</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9 Frames</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 Frames</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="rewindSummary">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
